app/helpers/application_helper/toolbar_chooser.rb

```diff
@@ -667,6 +667,7 @@ def unassigned_configuration_profile_node(nodes)
   end
 
   NO_GTL_VIEW_BUTTONS = %w(chargeback
+                           generic_object
                            generic_object_definition
```

Comment:
I am ok to remove it. I did not find why that value was there and did not dare removing it.